app/ui/browser/views/navbar/location.jsx

```diff
@@ -163,6 +163,17 @@ export class Location extends Component {
     return 'glyph-bookmark-hollow-16.svg';
   }
 
+  getVisibleCompletionsForURL() {
```

Comment:
Also, it feels like this might make sense as a selector rather than a function on the component.